runtime/src/lib.rs

```diff
@@ -351,6 +352,36 @@ parameter_type_with_key! {
 	};
 }
 
+type EnsureRootOrHalfCouncil = EnsureOneOf<
+	AccountId,
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, CouncilCollective>,
+>;
+
+parameter_types! {
+	pub const BasicDeposit: Balance = 20 * DOLLARS + 25_800 * MILLICENTS;
```

Comment:
WTF :D why it's not just 20 DOLLARS?

Reply:
Is this some kind of byte fee?

Reply:
I used the exact numbers. Ok. I will round the deposits to Dollars.

Reply:
I get the numbers from the `deposit()` method.

Reply:
I'm trying to understand where did the number come from. If it's something that was actually setup by weights then we should state it? If it's by using deposit method why is it not used here?

Reply:
I found two different implementations. 
One is using dollar amount: https://github.com/paritytech/polkadot/blob/a5c5512efb27edd149d3ae2a21a8b39adab7521a/runtime/kusama/src/lib.rs#L742
One is using deposit function: https://github.com/paritytech/polkadot/blob/a5c5512efb27edd149d3ae2a21a8b39adab7521a/runtime/polkadot/src/lib.rs#L420

It probably depends on what we want to achieve?

Reply:
We don't use this function so I substituted the function call with the result.

Reply:
So we decided to go with only dollar amount in the end.

Reply:
I got a question from our community managers about what's good amount to leave non-bound and unfortunately in a rush, I said that 100hdx will be ok not thinking about identity. Can we make this so that it fits that bill? I guess the space cost is 25.8 cents only, so it's still very safe to have 5 * DOLLARS here. It should then fit 100HDX with enough for some tx fees.